java/fury-core/src/main/java/io/fury/serializer/Serializers.java

```diff
@@ -429,38 +485,17 @@ public StringBuilder read(MemoryBuffer buffer) {
     }
   }
 
-  public static final class StringBufferSerializer extends Serializer<StringBuffer> {
-    private final StringSerializer stringSerializer;
+  public static final class StringBufferSerializer
+      extends AbstractStringBuilderSerializer<StringBuffer> {
 
     public StringBufferSerializer(Fury fury) {
       super(fury, StringBuffer.class);
-      stringSerializer = new StringSerializer(fury);
-    }
-
-    @Override
-    public short getXtypeId() {
```

Comment:
Can we add this to `AbstractStringBuilderSerializer`? Otherwise cross-language StringBuilder/Buffer will raise error?

Reply:
yes,i can to do